docs/blog/posts/welcome.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Welcome to the PyPI Blog
+author: Ee Durbin
+author_gh_user: ewdurbon
```

Comment:
```suggestion
author_gh_user: ewdurbin
```

Reply:
We're not using this so I just removed it.